crypto/ec/ecp_secp384r1.c

```diff
@@ -1,20 +1,19 @@
 /* Autogenerated: ECCKiila https://gitlab.com/nisec/ecckiila */
 /*-
  * MIT License
- *
- * Copyright (c) 2020 Luis Rivera-Zamarripa, Jesús-Javier Chi-Domínguez, Billy
- * Bob Brumley
- *
+ * 
+ * Copyright (c) 2020 Luis Rivera-Zamarripa, Jesús-Javier Chi-Domínguez, Billy Bob Brumley
```

Comment:
We're going to need ICLAs for everyone.

Reply:
> We're going to need ICLAs for everyone.

Done -- CLAs sent from @luinxz and @JJChiDguez

Reply:
Confirmed - all required CLAs are in place.